### DISCUSSION TITLE
Help with 1310 nm Grating Coupler

### DISCUSSION THREAD

**Discussion by PaoloZambon** [SiEPIC/SiEPIC_EBeam_PDK#296] [General]
I'm planning to use the 1310 nm grating coupler (TE polarization) from the SiEPIC Shuksan library (GC_TM_1310_8degOxide_BB).  I'd like to know if the hatched circle in the diagram represents the as-designed location of a fiber input into the grating.  I suppose one needs to assume propagation in a SiO2 layer, single mode, 1310 nm, TM polarization, 8 degrees angle. 

<img width="940" alt="GC_TM_1310_8degOxide_BB" src="https://github.com/user-attachments/assets/10e50ca1-4c0a-4fc8-a7d9-355a13fd7408">

**Comment by lukasc-ubc** [comment]
Hi,

The circle corresponds to the approximate location for the optical fibre.

Note that the black box cell components need to be replaced for fabrication.
